src/Swatch.scss

```diff
@@ -52,19 +52,27 @@ version of preview. Hopefully there will be a test of this diff at some point*/
 
 /*Basic layout swatches: moss, hdc, granny, shell, v-stitch*/
 @mixin interwovenStitch($stitchHeight, $stitchWidth, $spacing, $overlapRatio) {
-  &.staggered {
+  &.staggered:not(.colorSwallowed):not(.colorStretched) {
```

Comment:
I want to find a better way to do all this logic stuff, right now this might create a bug in non interwoven stitches that color stretch or swallow (but those don't actually exist?)